File: C & C++ PRACTICE CODES/Triangle Pattern/code1.c

#include <stdio.h>

void main (){

	int row;
	printf("enter no. of row : ");
	scanf("%d",&row);

	for(int i = 1; i <= row ; i++){

		for(int j = row ; j < row + i ; j++){


			printf("\t%d\t",j +  1 - row);

		}
		printf("\n\n");

	}

}

File: JAVA DAILYCODES ASSGNPRATICE QUIZE DSA/Basic is else assignments/9/code1.java

import java.util.Scanner;

class NestedDemo{

	public static void main(String[] args){

		Scanner s= new Scanner(System.in);
		System.out.print("Enter no of rows : ");
		int row = s.nextInt();

		for(int i = row; i > 0 ; i--){

			for(int j = 0; j < row ; j++){

				System.out.print((char)(64+i)+"		");

			}

				System.out.print("\n\n");

		}

	}

}
